Script.md

```diff
@@ -0,0 +1,27 @@
+## Introducing Lodgel – the property management application that streamlines the rental process.
+Lodgel removes the hassle of ensuring your properties are ready for guests by coordinating assistants
+between properties and customizing checklists with tasks based on the needs of your guests. And if your guests
+have a special request? No problem. The checklists can be updated anytime something else is needed, and you can 
+check in on your assistants to ensure the job is done on time. We even integrated Stripe - one of the world's leading
+payment processors, so you can get paid as soon as your guests check out.

```

Comment:
For an elevator pitch this is too long & misses features.

This is going to be incredibly important as it sets the tone for the whole video and will play a huge part if the video is received positively or  not.

Reply:
This is just a rough idea, but maybe you can run with it: 

Welcome to Lodgel!
Our aim is, to take the hassle out of out of your property management experience.
You can quickly delegate tasks to your turnover assistants, conveniently schedule guests and securely capture their payments at the click of a button, all this while enjoying the support of our great team.

Reply:
✔️